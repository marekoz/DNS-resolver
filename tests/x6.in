-s 1.1.1.1 -r -x 2a00:1450:4014:080f:0000:0000:0000:200e