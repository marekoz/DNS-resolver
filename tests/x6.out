Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
  e.0.0.2.0.0.0.0.0.0.0.0.0.0.0.0.f.0.8.0.4.1.0.4.0.5.4.1.0.0.a.2.ip6.arpa., Type: PTR, Class: IN
Answer section (1)
  e.0.0.2.0.0.0.0.0.0.0.0.0.0.0.0.f.0.8.0.4.1.0.4.0.5.4.1.0.0.a.2.ip6.arpa., Type: PTR, Class: IN, TTL: <number>, Data length: 27, prg03s13-in-x0e.1e100.net.
Authority section (0)
Additional section (0)